### DISCUSSION TITLE
CSS and Scripts lost when restoring versions

### DISCUSSION THREAD

**Discussion by clivey-za** [requarks/wiki#6524] [Error / Bug Report] [answered]
Hi all,

When creating a raw html page with scripts and css, if history is used to restore versions then scripts and css are always completely cleared - even when they existed before the date restored.

Using 2.5.298

**Comment by NGPixel** [comment, marked as answer]
CSS and Scripts are not versioned at the moment. Will be addressed in a future version.

**Reply by aboqasem** [reply]
Hello @NGPixel, has this been fixed? It still seems to be the case. Also, any chance a global script can be added to ALL pages from the admin dashboard? Thanks!

**Reply by NGPixel** [reply]
You can already do so from Admin > Theme
